### ISSUE TITLE
paratype font conflict with preset setting

### ISSUE THREAD

**Issue by ghost** [drenninghoff/infinality-ultimate-fedora#16]
there is a paratype font which provide "PT Sans" typeface, and by choosing any preset font configuration by `fc-preset` command, this font always became default sans font for system:

``` bash
$ fc-match sans
PTS55F.ttf: "PT Sans" "Regular"
```

**Comment by danielrenninghoff** [comment]
Yes, you are right. I will create a package for paratype-pt-sans, but there is no solution other than manually removing the fedora package (which gets installted automatically on Fedora 24) :(.
